db/migrate/20190301174502_remove_rss_feeds.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveRssFeeds < ActiveRecord::Migration[5.0]
+  def change
+    drop_table :rss_feeds
```

Comment:
The cop is complaining because this isn't reversible?  If we don't want to put the table in the down migration, maybe we should write an `up` without a `down`.... but why not just recreate it as it is now in the `down`?

Reply:
The reverse is required.